packages/docs/src/components/Layout.tsx

```diff
@@ -57,7 +57,7 @@ const Layout = ({
   return (
     <div className="ds-base">
       <Helmet
-        title="CMS Design System"
+        title={`${frontmatter?.title || ''} - CMS Design System`}
```

Comment:
Could this be rearranged so that if there's no title it shows `CMS Design System` instead of `- CMS Design System`? Or is it never the case that a page will not have a title and this is just to appease TypeScript?

Reply:
I don't think a page would every be without a title, but I did update this, just in case